client/public/locales/en/translation.json

```diff
@@ -265,6 +265,8 @@
     "applications": "Applications",
     "archetype": "Archetype",
     "archetypes": "Archetypes",
+    "archetypes_singular": "Archetype",
```

Comment:
We should really upgrade our i18next version so this key structure is correct (and matches current docs).  IIRC for the current version in use, the singular key should not have "_singular" appended.